src/main/java/se/liu/ida/hefquin/engine/queryproc/impl/optimizer/evolutionaryAlgorithm/EvolutionaryAlgorithmQueryOptimizer.java

```diff
@@ -59,7 +59,6 @@ public EvolutionaryAlgorithmQueryOptimizer( final QueryOptimizationContext ctxt,
 
         final QueryOptimizationStatsImpl myStats = new QueryOptimizationStatsImpl();
         myStats.put( "numberOfGenerations", previousGenerations.size() + 1 );
-        myStats.put( "costOfSelectedPlan", bestPlan.getWeight() );
```

Comment:
I think you don't need to delete this. It doesn't hurt to have this stats entry here too, and here it is always added, not only during experiment runs.

Reply:
You're right, it is not only added during experiment runs. Sorry I didn't think of this.